### DISCUSSION TITLE
Nuxt 3.3.3 build error after updating node.js to latest version

### DISCUSSION THREAD

**Discussion by Usman024** [nuxt/nuxt#27460] [Questions]
I was working on nuxt version 3.3.3 and I have to update node to latest version 20 for some other project
But after updating node i did npm run dev in nuxt 3.3.3 version and it wasn't building

Now I installed nvm and switched back to node 18.16.0 I am not sure I was using this before 🥲

it's working but with some breaking in the project 
Can someone please assist me 🙏

**Comment by chibx** [comment]
@Usman024 Is there a link to reproduction, or could you show us the error message itself
